Projects/TEAM-2/GroupProject/GroupProject/Customer.cs

```diff
@@ -7,7 +7,31 @@
 
 namespace GroupProject
 {
-    public class Customer
+    public class Customers
     {
+       public string Name;
+       public float money;
+
+       public Customers(string Name, float money)
+        {
+            this.Name = Name;
+            this.money = money;
+        }
+       
+    public string Set_Name(string Name)
+        {
+            this.Name = Name;
+
+            return Name.ToString();
+        }
+
+    public float Set_money(float money)
+        {
+            this.money = money;
+
+            return money.ToString();
+        }
+
+     Customers new_customer = new Customers();
```

Comment:
It's better to create customers in the Program class. Delete or comment this line.

Reply:
Complete